### PR TITLE
Young inspirers page

### PR DESCRIPTION
**Pull request by SivanYeh**
## Types of changes

* **New feature**

## Steps to Test This Pull Request

1. Click on "Conference"/"young-inspirers" on navigation bar.
2. Scroll down and see if speech boxes can be seen.
3. Click on a speech box and see if modal pops up with further details.

## Expected behavior

image
## Related Issue

#258

### CODE REVIEW

**Review by josix — COMMENTED** [review]
Thank you @SivanYeh for the hard work, I've updated this page to follow the spec's styling and added the latest information.

File: pages/conference/young-inspirers.vue [hunk 1]
```diff
@@ -1,266 +1,236 @@
...
+                ],
+                speech4: [require('~/static/img/young-inspirers/speaker4.svg')],
+            },
+            tagToSpeaker: {
+                speech1: this.$i18n.t('speech1Speaker'),
+                speech2: this.$i18n.t('speech2Speaker'),
+                speech3: this.$i18n.t('speech3Speaker'),
+                speech4: this.$i18n.t('speech4Speaker'),
+            },
+            tagToLangIcon: {
```
Comment (mattwang44) [inline pages/conference/young-inspirers.vue:199]: the style of icons should conform to the spec (may need designer's help for the zh icon)

image

File: pages/conference/young-inspirers.vue [hunk 2]
```diff
@@ -1,266 +1,236 @@
...
+                                        class="linkButton"
+                                    >
+                                        觀看直播
+                                    </text-button>
+                                    <text-button
+                                        href="speech.note_link"
+                                        secondary
+                                        large
+                                        class="linkButton"
+                                    >
```
Comment (mattwang44) [inline pages/conference/young-inspirers.vue:90]:
```suggestion
                                    <text-button
                                        :href="speech.live_link"
                                        small
                                        class="linkButton"
                                    >
                                        觀看直播
                                    </text-button>
                                    <text-button
                                        :href="speech.note_link"
                                        secondary
                                        large
                                        class="linkButton"
                                    >
```

**Review by mattwang44 — CHANGES_REQUESTED** [review]

File: i18n/conference/young-inspirers.i18n.js [hunk 3]
```diff
@@ -2,15 +2,199 @@ import { genI18nMessages } from '~/utils/i18n.utils'
...
+                        full_description:
+                            '前半段分享我們兩人在高中階段的公益社團、演講、比賽及學習Python的過程。後面將以黃金圈理論做收斂及整理，跟大家分享如何找到自己的人生目標(Why)，朝向目標的方向與方法(How)，在每一個階段可以做的事情(What)。如果大學生活會改變一個人的價值，如何在進大學之前先學會不虛度光陰，學會自我學習的方式，讓每一天都朝向自己的人生目標前進。',
+                        note_link: '',
+                        live_link: 'https://youtu.be/SAbYVx_7okE',
+                    },
+                    {
+                        tag: 'speech4',
+                        speechdate: 'July 30',
+                        speechtime: '15:00-15:40',
+                        title: 'Comeing S',
```
Comment (mattwang44) [inline i18n/conference/young-inspirers.i18n.js:185]: typo?

File: pages/conference/young-inspirers.vue [hunk 4]
```diff
@@ -1,266 +1,236 @@
...
+                                        v-for="(speaker, i) in tagToSpeaker[
+                                            speech.tag
+                                        ]"
+                                        :key="`speech_info_${speechIdx}_speaker_${i}`"
+                                    >
+                                        {{ speaker }}
+                                    </span>
+                                </div>
+                                <p class="speechBox__description">
+                                    {{ speech.full_description }}
```
Comment (mattwang44) [inline pages/conference/young-inspirers.vue:75]: The description should be truncated if it's too long. The following is the snapshot of figma.

image

**Review by mattwang44 — CHANGES_REQUESTED** [review]
Mobile view issues

File: pages/conference/young-inspirers.vue (hunk 4 above)
Comment (mattwang44) [reply pages/conference/young-inspirers.vue:75]: spec deleted

**Review by mattwang44 — APPROVED** [review]
I've rebased the branch and roughly cleaned up the commit history.
Please open another PR for following issues (if any).

**Pull request merged**